docs/learning/geometry/Angle.md

```diff
@@ -1,8 +1,8 @@
 ## fraction of an angle sweep returned by different API
-![>](./figs/Angle/angleSweepFraction.jpg)
+![>](./figs/Angle/AngleSweepFraction.jpg)
```

Comment:
What was the motivation for this change? Filenames are case-sensitive on some platforms and the case here no longer matches.

Reply:
That's because I noticed all image names under docs/learning/geometry are CamelCase expect these 2 files

Reply:
Do you intend to rename the files, or to revert the changes to the markdown?